README.md

```diff
@@ -374,13 +376,43 @@ let app =
         ]
 ```
 
+### subRoute
+
+`subRoute` checks the current path begins with the `path` string, all `route*` and `subRoute*` handlers within the `handler` parameter will now be rooted to, and relative with respect to this path.
+
+#### Example:
+
+```fsharp
+let app = 
+    subRoute "/api" >>=
+        choose [
+            route "/v1/foo" >>= text "Foo"
+            route "/v1/bar" >>= text "Bar"
+        ]
+```
+
```

Comment:
There was a typo in my example, we don't use the `>>=` operator, as that is done within `subRoute`, so I'll take and update your suggestion